src/pages/office/cat-cpc/office.cat-cpc.page.html

```diff
@@ -33,36 +43,59 @@ <h4>Test summary</h4>
 
           <div no-padding>
 
-            <candidate-description [display]="pageState.displayCandidateDescription$ | async" [formGroup]="form"
-                                   [candidateDescription]="pageState.candidateDescription$ | async"
-                                   [outcome]="pageState.testOutcome$ | async"
-                                   (candidateDescriptionChange)="candidateDescriptionChanged($event)">
+            <language-preferences
+              *ngIf="isDelegated"
+              [isDelegated]="isDelegated"
+              [formGroup]="form"
+              [isWelsh]="isWelsh()"
+              (welshChanged)="isWelshChanged($event)"></language-preferences>
+
+            <debrief-witnessed
+              [display]="isDelegated"
+              [isDelegated]="isDelegated"
+              [formGroup]="form"
+              [debriefWitnessed]="pageState.debriefWitnessed$ | async"
+              [outcome]="pageState.testOutcome$ | async"
+              (debriefWitnessedChange)="debriefWitnessedChanged($event)">
+            </debrief-witnessed>
+
+            <candidate-description
+              *ngIf="!(isDelegated && !isPass())"
+              [display]="(pageState.displayCandidateDescription$ | async)"
+              [formGroup]="form"
+              [candidateDescription]="pageState.candidateDescription$ | async"
+              [outcome]="pageState.testOutcome$ | async"
+              (candidateDescriptionChange)="candidateDescriptionChanged($event)">
             </candidate-description>
 
-            <identification [display]="pageState.displayIdentification$ | async" [formGroup]="form"
-                            [identification]="pageState.identification$ | async"
-                            [outcome]="pageState.testOutcome$ | async"
-                            (identificationChange)="identificationChanged($event)">
+            <identification
+              *ngIf="!(isDelegated && !isPass())"
```

Comment:
is !isPass here correct? doesnt look like its on any of the designs for any of the outcomes (pass, fail, term)